### PR TITLE
Better padding with init command sample files.

### PR DESCRIPTION
**Pull request by winder**
## Summary

The importer/exporter sample files required proper padding, while the processor samples were required to be un-padded.
With this change all sample files may be un-padded. Any accidental padding will still work, and only causes the config file to look strange rather than resulting in a broken file.

With these changes we also consistently pad with 4 spaces. Before there was a mixture of 2/4 space indents.

## Test Plan

Manual testing.

### CODE REVIEW

File: pkg/cli/internal/initialize/init.go [hunk 1]
```diff
@@ -27,12 +27,11 @@ var errNoWriter = errors.New("configWriter is required")
 //go:embed conduit.yml.example
 var sampleConfig string
 
-func formatArrayObject(obj string) string {
-
+func formatObject(obj string, isArray bool) string {
```
Comment (winder) [inline pkg/cli/internal/initialize/init.go:30]: The main changes is to use this function for each type of configuration. Once that was done, I updated each of the sample files to use consistent padding.

File: conduit/plugins/exporters/postgresql/sample.yaml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-      interval: 0
-
-      # Rounds to keep
-      rounds: 100000
+        # Interval used to prune the data. The values can be -1 to run at startup,
+        # 0 to disable, or N to run every N rounds.
+        interval: 0
+    
+        # Rounds to keep
+        rounds: 100000
```
Comment (tzaffi) [inline conduit/plugins/exporters/postgresql/sample.yaml:20]:
```suggestion
        rounds: 100000

```

File: conduit/plugins/importers/algod/sample.yaml [hunk 3]
```diff
@@ -27,4 +27,4 @@
         #   mainnet: https://algorand-catchpoints.s3.us-east-2.amazonaws.com/consolidated/mainnet_catchpoints.txt
         #   betanet: https://algorand-catchpoints.s3.us-east-2.amazonaws.com/consolidated/betanet_catchpoints.txt
         #   testnet: https://algorand-catchpoints.s3.us-east-2.amazonaws.com/consolidated/testnet_catchpoints.txt
-        catchpoint: ""
+        catchpoint: ""
```
[the remark this answers is not in the document]
Comment (tzaffi) [reply conduit/plugins/importers/algod/sample.yaml:30]: See also:
* `conduit/plugins/processors/filterprocessor/sample.yaml`
* `conduit/plugins/processors/noop/sample.yaml`
* `pkg/cli/internal/initialize/conduit.yml.example`

[the remark this answers is not in the document]
Comment (winder) [reply conduit/plugins/importers/algod/sample.yaml:30]: Removing them made the spacing look better on the generated `conduit.yaml`. Why do you like an extra newline at the end?

**Review by Eric-Warehime — APPROVED** [review]

**Review by tzaffi — APPROVED** [review]

**Pull request merged**